package.json

```diff
@@ -33,6 +33,7 @@
   },
   "homepage": "https://github.com/fedor/node-no-config#readme",
   "dependencies": {
-    "co": "^4.6.0"
+    "co": "^4.6.0",
+    "tape": "^4.6.0"
```

Comment:
@anuragCES good. One thing I've mentioned here before is a need to keep `tape` as a **dev** dependency because it's only used for its testing.

Please check this link: https://docs.npmjs.com/files/package.json#devdependencies

```json
"dependencies": {
    ...
},
"devDependencies": {
    ...
}
```